charts/prometheus/values.yaml

```diff
@@ -348,13 +348,15 @@ alertmanager:
   ## Security context to be added to alertmanager pods
   ##
   securityContext:
+    enabled: true
     runAsUser: 65534
     runAsNonRoot: true
     runAsGroup: 65534
     fsGroup: 65534
 
   ## Security context to be added to alertmanager container
-  containerSecurityContext: {}
+  containerSecurityContext:
+    enabled: false
```

Comment:
I'm not sure this is right.
Because there are some people who already have some parameters in the containerSecurityContext, this is a breaking change.
This will force them to have to add enabled: true everywhere.
I'd suggest you do it the other way, and have the people who don't want a SecurityContext set it to false.